code/modules/detectivework/microscope/microscope.dm

```diff
@@ -2,6 +2,8 @@
 /obj/machinery/microscope
 	name = "high powered electron microscope"
 	desc = "A highly advanced microscope capable of zooming up to 3000x."
+	desc_info = "Use a microscope slide or a fingerprint card on this machine to insert it.\
+	\nAlt click to remove any object within it."
```

Comment:
This looks a little funny. This is just a linebreak right?

Reply:
Yes. I added them to the descriptions to make it easier to read.